### DISCUSSION TITLE
Plugins don't work at all.

### DISCUSSION THREAD

**Discussion by DELTASz** [IzzelAliz/Arclight#1092] [General]
After the first reboot of the server, the plugins do not even try to start, because the mods work quietly. The logs are completely empty, there are only logs about launching mods. (1.19.2-1.0.2)

**Comment by wrldmap** [comment]
Are you using the Forge run.bat/run.sh? You need to start the Arclight jar file. I know it gets confusing.

**Reply by Imjust1persn** [reply]
THIS SOLVED EVERYTHING THANK YOU
